vars/processTemplate.groovy

```diff
@@ -36,6 +36,9 @@ def applyDefaults(provided=[:], templateParams) {
     if (key in templateParams) {
       params[key] = provided[key] ?: compute()
     }
+    else{
```

Comment:
have you tested this? wouldn't this add param even if it isn't present in the template parameters which would cause oc process to fail?

Reply:
Exactly what happened https://github.com/fabric8io/osio-pipeline/issues/64#issuecomment-436978237

Haven't tested `--ignore-unknown-parameters` though.

Reply:
It is better to not ignore-unkown-params as it is going to fail silently which makes it harder to debug.